src/main/docker/Dockerfile

```diff
@@ -71,33 +71,31 @@ RUN dpkg-reconfigure --terse --frontend noninteractive locales
 
 # Set up our configuration files
 COPY docker-entrypoint.sh /usr/local/bin/
-COPY configs/"${project.artifactId}.properties.tmpl" /etc/"${project.artifactId}/${project.artifactId}.properties.tmpl"
-COPY configs/"${project.artifactId}.properties.default" /etc/"${project.artifactId}/${project.artifactId}.properties.default"
-COPY configs/logback.xml /etc/"${project.artifactId}"/logback.xml
+COPY configs/bucketeer.properties.tmpl /etc/bucketeer/bucketeer.properties.tmpl
+COPY configs/bucketeer.properties.default /etc/bucketeer/bucketeer.properties.default
+COPY configs/logback.xml /etc/bucketeer/logback.xml
 
 # Create a non-privileged user
-RUN adduser --system "${project.artifactId}"
-WORKDIR /home/"${project.artifactId}"
+RUN addgroup --gid 1000 bucketeer && \
+    adduser --system --uid 1000 --gid 1000 --no-create-home --disabled-login --disabled-password bucketeer
```

Comment:
I'm wondering if we should pick a UID that is less common. `1000` is typically the first user to be created on a fresh system. It might not necessarily be a problem, but it might conflict with UIDs on mounted volumes. Though I don't think we would have a problem in our bucketeer app. This is more of a suggestion.

Reply:
I did wonder whether I should go back to using the `--system` arg since I'm not trying to mount anything any more. That uses the 101 uid. I kept it because I'd turned off home dir and password, etc., but that might be possible with the system user too? Thoughts?

Reply:
the `--system` arg does make it seem a bit confusing, those generally range from 100-499. I'm fine with this as a solution. It has the same functionality.